apps/site/public/demo-utils/touchai-lite-renderer.js

```
@@ -0,0 +1,102 @@
+
+        return segments
+            .map((segment) => {
+                if (!segment) return '';
+                if (segment.startsWith('`') && segment.endsWith('`')) {
+                    return `<span class="kbd">${escapeHtml(segment.slice(1, -1))}</span>`;
+                }
+
+                return escapeHtml(segment).replace(
+                    /\*\*([^*]+)\*\*/g,
+                    '<strong>$1</strong>'
+                );
```

ESLint

[error] Replace `⏎····················/\*\*([^*]+)\*\*/g,⏎····················'<strong>$1</strong>'⏎················` with `/\*\*([^*]+)\*\*/g,·'<strong>$1</strong>'`

(prettier/prettier)